authors/apps/articles/slug_creator.py

```diff
@@ -0,0 +1,21 @@
+from django.template.defaultfilters import slugify
```

Comment:
Please rename this file to something generic like `utilities.py` incase we need to add more functions that are required in more than one place. You could also move it to `authors/apps` such that we are able to access it across different apps.